CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-# dbt_social_media_reporting version.version
+# dbt_social_media_reporting v0.6.0
+[PR #16](https://github.com/fivetran/dbt_social_media_reporting/pull/16) includes the following updates:
+
+## Breaking Change: Addition of Youtube Analytics
+- We have added the Youtube Analytics as a platform in this package via the [dbt_youtube_analytics](https://github.com/fivetran/dbt_youtube_analytics) package. If you are not using the Youtube Analytics connection, you may disable it by following [these steps in the README](https://github.com/fivetran/dbt_social_media_reporting/blob/main/README.md#step-4-enablingdisabling-models) or by including the configuration shown below in your `dbt_project.yml` file:
```

Comment:
Perhaps we should note that we're specifically bringing in the `youtube__video_report` model

Reply:
added!